src/main/java/org/fundacionjala/at15/katas/langtonant/gonzalo/Grid.java

```diff
@@ -0,0 +1,45 @@
+package org.fundacionjala.at15.katas.langtonant.gonzalo;
+
+public class Grid {
+    private int wide, high;
+    private char [][] grid;
+    
+    public Grid(int wide, int high){
+        this.wide=wide;
+        this.high=high;
+        grid=new char[wide][high];
+    }
+    
+    public void loadGrid(){       
+        for(int column=0; column<high;column++){
+            for(int file=0; file<wide;file++){
+                grid[column][file]='█';
+            }
+        }
+    }
+    
+    public void printGrid(){
+        for(int column=0; column<high;column++){
+            for(int file=0; file<wide;file++){
+                System.out.print(grid[column][file]+" ");
+            }
+            System.out.println();
+        }
+    }
+    
+    public int getPosition(int posX, int posY){
+        return grid[posX][posY];
+    }
+    
+      
+    public void changeValue(int posX, int posY){
+        if(grid[posX][posY]=='█'){
```

Comment:
It seems that there might be some encoding issues with your editor. Make sure you are using UTF-8

Reply:
If using VS Code, you can change encoding like this:
![image](https://user-images.githubusercontent.com/4606607/153331837-ae415855-520a-4e8a-b444-b1e38df281be.png)